packages/integration-sdk-cli/src/generator/entitiesFlow.ts

```diff
@@ -0,0 +1,190 @@
+import { askAgain } from './util';
+import { snakeCase } from 'lodash';
+
+export type Entity = {
```

Comment:
Is it possible to share the entity type? Perhaps you can use `Partial` to indicate the properties that are necessary for this file's use cases.